test/controllers/trails_controller_test.rb

```diff
@@ -1,23 +1,35 @@
 require 'test_helper'
 
-class Api::V1::Trails::TrailsControllerTest < ActionController::TestCase
+class Api::V1::TrailsControllerTest < ActionController::TestCase
 
   test "should get trails index" do
-    get :index
+    get :index, format: :json
 
     data = JSON.parse(response.body)
-    
+
     assert_response :success
     assert_kind_of Hash, data
   end
 
   test "should get trails show" do
-    trail = Trail.create(name: "howdy hike", city: "Denver", state: "CO", country: "United States", lat: 10.323, lng: -103.23, description: "A darn good time", directions: "take a right" )
-    get :show, id: trail.id
+    trail = Trail.create(name: "howdy hike", city: "Denver", state: "CO", country: "United States", lat: 10.323, lng: -103.23, description: "A darn good time", directions: "take a right")
+    get :show, id: trail.id, format: :json
 
     data = JSON.parse(response.body)
-    
+
     assert_response :success
     assert_kind_of Hash, data
   end
+
+  test "trails have nested activities" do
+    trail = Trail.create(name: "howdy hike", city: "Denver", state: "CO", country: "United States", lat: 10.323, lng: -103.23, description: "A darn good time", directions: "take a right")   
```

Comment:
Line is too long. [190/80]<br>Unnecessary spacing detected.<br>Trailing whitespace detected.